### ISSUE TITLE
Show start/end times on each step in the build

### ISSUE THREAD

**Issue by cppforlife** [concourse/concourse#428]
doesn't seem time offsets (or full timestamps) are recorded right now but it would be great to have those times on each task next names to easily see how long each step takes.

**Comment by JesseTAlford** [comment]
We've seen an issue where we think tasks are taking a very long time to start doing anything, but it's hard to produce evidence for/trace because there's no per-task time information.

In general, people are often concerned with how long things take in CI. The better the resolution we can get on this, the better.

**Comment by jchesterpivotal** [comment]
We're encountering this need on buildpacks team also. Many of our tasks are long-running, but may also have slow `get` steps. Being able to distinguish will allow us to decide when a long-running task has gone feral.

It's difficult to tell, because we can't subtract the `task` time or the `get` time, because they are not individually timed.

More generally, this will affect *anyone* using resource pools. When a lock takes a long time to become available, total build time is inflated. While total time is a hint, I for one am terrible at taking hints.

**Comment by matt-royal** [comment]
This would also be helpful for the Ops Manager team, for the reasons @JesseTAlford gave.

**Comment by jraqula** [comment]
:+1: BOSH would like to see more detail about which steps are taking longer than usual in our long test cycles.

**Comment by mcwumbly** [comment]
It would be great to have this information shown in the UI.  It's amazing how much time teams waste because of slow feedback cycles on builds.  It'd be a lot easier to improve pipeline efficiency with more feedback about how long things actually take.

**Comment by marco-m** [comment]
I agree with all the comments in this thread, people often get frustrated due to the lack of visibility of why nothing seems to be happening, so having better timing information would be a very welcomed enhancement.

To share some good news with my fellow commenters, we deployed Concourse 5 RC since a few weeks and we enabled the "put inputs" as described here: #1202. The majority of inexplicable slownesses went away.
